plugins.txt

```diff
@@ -1,137 +1,119 @@
+blueocean-pipeline-api-impl:1.4.2
+blueocean-pipeline-editor:1.4.2
+blueocean-pipeline-scm-api:1.4.2
+blueocean-rest-impl:1.4.2
+blueocean-rest:1.4.2
+blueocean-web:1.4.2
+blueocean:1.4.2
 bouncycastle-api:2.16.2
-branch-api:2.0.16
+branch-api:2.0.18
 build-timeout:1.19
-cloudbees-bitbucket-branch-source:2.2.7
-cloudbees-folder:6.2.1
-config-file-provider:2.16.4
-credentials-binding:1.13
+cloudbees-folder:6.3
+config-file-provider:2.18
+credentials-binding:1.15
 credentials:2.1.16
 display-url-api:2.2.0
-docker-commons:1.9
-docker-workflow:1.14
-dockerhub-notification:2.2.0
-durable-task:1.17
+docker-commons:1.11
+docker-workflow:1.15.1
+dockerhub-notification:2.2.1
+durable-task:1.21
 email-ext:2.61
 embeddable-build-status:1.9
 external-monitor-job:1.7
 favorite:2.3.1
-ghprb:1.39.0
-git-client:2.6.0
+ghprb:1.40.0
+git-client:2.7.1
 git-server:1.7
-git:3.6.4
+git:3.8.0
 github-api:1.90
-github-branch-source:2.3.1
+github-branch-source:2.3.2
 github-issues:1.2.4
-github-oauth:0.28.1
+github-oauth:0.29
 github-organization-folder:1.6
 github-pr-coverage-status:1.9.1
-github-pullrequest:0.1.0-rc27
-github:1.28.1
-gitlab-merge-request-jenkins:2.0.0
-gitlab-oauth:1.0.9
-gitlab-plugin:1.5.2
-google-login:1.3
+github-pullrequest:0.1.0-rc29
+github:1.29.0
 gradle:1.28
-gravatar:2.1
-handlebars:1.1.1
 htmlpublisher:1.14
 icon-shim:2.0.3
-jackson2-api:2.8.7.0
-jira:2.5
+jackson2-api:2.8.11.1
 jquery-detached:1.2.1
 jsch:0.1.54.1
-junit:1.23
+junit:1.24
 kerberos-sso:1.4
-kubernetes:1.1.2
-ldap:1.18
-lockable-resources:2.1
+kubernetes:1.3.2
+lockable-resources:2.2
 mailer:1.20
 mapdb-api:1.0.9.0
 mask-passwords:2.10.1
 matrix-auth:2.2
 matrix-project:1.12
-mercurial:2.2
-metrics:3.1.2.10
+metrics:3.1.2.11
 momentjs:1.1.1
-multiple-scms:0.6
-nodejs:1.2.4
+nodejs:1.2.5
 oauth-credentials:0.3
-oic-auth:1.0
+oic-auth:1.2
 openid:2.2
 openid4java:0.9.8.0
 openshift-client:1.0.2
 openshift-pipeline:1.0.52
 openshift-sync:0.1.32
 pam-auth:1.3
-pipeline-build-step:2.5.1
+pipeline-build-step:2.7
 pipeline-github-lib:1.0
-pipeline-githubnotify-step:1.0.3
-pipeline-graph-analysis:1.5
+pipeline-githubnotify-step:1.0.4
+pipeline-graph-analysis:1.6
 pipeline-input-step:2.8
 pipeline-milestone-step:1.3.1
-pipeline-model-api:1.2.5
+pipeline-model-api:1.2.7
 pipeline-model-declarative-agent:1.1.1
-pipeline-model-definition:1.2.5
-pipeline-model-extensions:1.2.5
+pipeline-model-definition:1.2.7
+pipeline-model-extensions:1.2.7
 pipeline-rest-api:2.9
 pipeline-stage-step:2.3
-pipeline-stage-tags-metadata:1.2.5
+pipeline-stage-tags-metadata:1.2.7
 pipeline-stage-view:2.9
-pipeline-utility-steps:1.5.1
+pipeline-utility-steps:2.0.1
 plain-credentials:1.4
 pubsub-light:1.12
 resource-disposer:0.8
-scm-api:2.2.5
-script-security:1.36
+scm-api:2.2.6
+script-security:1.42
 sse-gateway:1.15
 ssh-agent:1.15
 ssh-credentials:1.13
-ssh-slaves:1.22
-structs:1.10
-subversion:2.9
-timestamper:1.8.8
+ssh-slaves:1.26
+structs:1.14
+timestamper:1.8.9
 token-macro:2.3
 updatebot:1.0.10
```

Comment:
Is this really needed? Do we use this bot anywhere?

Reply:
No idea, I have just tried to remove what I know we really don't need it.

Reply:
I doubt our repos are using it and that should be easy to check. If we make it available for the users we have to take care of updates / fixes. https://github.com/fabric8-updatebot/updatebot